modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectAuthenticationHandlerImpl.java

```diff
@@ -137,8 +138,34 @@ public void processAuthenticationResponse(
 			_getLoginRedirectURI(httpServletRequest),
 			oAuthClientEntry.getTokenRequestParametersJSON());
 
-		String userInfoJSON = _requestUserInfoJSON(
-			oidcTokens.getAccessToken(), oidcProviderMetadata);
+		URI userInfoURI = oidcProviderMetadata.getUserInfoEndpointURI();
+
+		String userInfoJSON = "";
```

Comment:
You can initialize this to null and avoid GC. Both if blocks will initialize it propertly.

Reply:
solved in the next commit...